client/test/consumerPact.spec.js

```diff
@@ -64,9 +65,14 @@ describe('Pact for Film Provider', () => {
         it('returns all films', () => {
             return filmService.getAllFilms()
                 .then(response => {
-                    expect(response).to.deep.members([new Film(1, "Star Wars", "Space", 1980), 
-                                                      new Film(2, "Superman", "Comic", 1986),
-                                                      new Film(10, "Indiana Jones", "Adventures", 1985)]);
+                    // Reason: 'like' matcher for id
+                    response.forEach(function(film) {
+                        expect(film.id).to.be.an('number');
+                    });
+                    response.forEach(function(film) { film.id = ANY_NUMBER; });
```

Comment:
Hi, Thanks for your PR.

I'm not quite sure about this substitution. I know that this test is just for creating the pact, but I'm not a big fan of changing the response to make the test passes.

What do you think?. Do you have any other alternative?

Reply:
Hi! I absolutely agree. I also didn't like it. Now, I realized that it's not necessary, because it seems 'like' matcher makes more sense in producer side, so I'm going to remove the last commit in which I made this "trick" :sweat_smile:  
Thanks!

Reply:
Done!! :wink: